evalml/preprocessing/data_splitters/balanced_classification_sampler.py

```diff
@@ -50,19 +50,19 @@ def _find_ideal_samples(self, y):
         """
         counts = y.value_counts()
         normalized_counts = y.value_counts(normalize=True)
-        class_ratios = normalized_counts / min(normalized_counts)
-        # if no class ratios are greater than what we consider balanced, then the target is balanced
-        if all(class_ratios <= self.balanced_ratio):
+        class_ratios = min(normalized_counts) / normalized_counts
+        # if no class ratios are larger than what we consider balanced, then the target is balanced
+        if all(class_ratios < self.balanced_ratio):
             return {}
         # if any classes have less than min_samples counts and are less than min_percentage of the total data,
         # then it's severely imbalanced
         if any(counts < self.min_samples) and any(normalized_counts < self.min_percentage):
             return {}
         # otherwise, we are imbalanced enough to perform on this
-        undersample_classes = counts[class_ratios > self.balanced_ratio].index.values
+        undersample_classes = counts[class_ratios <= self.balanced_ratio].index.values
         # find goal size, round it down if it's a float
         minority_class = min(counts.values)
-        goal_value = max(int((self.balanced_ratio * minority_class) // 1), self.min_samples)
+        goal_value = max(int((minority_class / self.balanced_ratio) // 1), self.min_samples)
```

Comment:
Makes sense! Since this PR has made it so that `balanced_ratio_new = 1 / balanced_ratio_old`, we just flip `self.balanced_ratio` over the dividing line. Nice.